engine/src/main/java/io/camunda/zeebe/engine/state/migration/DbMigrationState.java

```diff
@@ -351,4 +351,10 @@ public void migrateElementInstancePopulateProcessInstanceByDefinitionKey() {
           }
         });
   }
+
+  @Override
+  public void markMigrationFinished(final String identifier) {
+    migrationIdentifier.wrapString(identifier);
+    migrationStateColumnFamily.upsert(migrationIdentifier, new MigrationState());
```

Comment:
🔧 I think we can use insert here if we're a bit smart about when we invoke this method

💡 I wrote about that in another comment below.

Reply:
True, we have to be careful once we introduce other states. But I suppose we don't have to worry about this now.